### PR TITLE
Localization issues

### PR DESCRIPTION
**Pull request by nguyen-binh-minh**
### Purpose

-  Localize Select node's button ([MAGN-7708](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7708))

deu_select_button

- Replace CR in tooltips ([MAGN-7698](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7698), [MAGN-7694](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7694))

cr

- Localize tooltips for custom node input Name and description ([MAGN-7712](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7712))
esp

- Localize "Convert between Units" node ([MAGN-7551](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7551))

28deu 29convert between units2

- Localize error messages in ProtoCore [MAGN-7721](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7721)

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] The level of testing this PR includes is appropriate

### Reviewers

@ke-yu PTAL

### CODE REVIEW

File: src/DynamoCore/Properties/Resources.en-US.resx [hunk 1]
```diff
@@ -286,7 +286,9 @@ Sorry about that.</value>
     <value>Loading node definition for "{0}" from: {1}</value>
   </data>
   <data name="NodeProblemEncountered" xml:space="preserve">
```
Comment (ke-yu) [inline src/DynamoCore/Properties/Resources.en-US.resx]: `NodeProblemEncountered` has a placeholder, as we remove it, shall we update its references to string concatenation?

**Comment by nguyen-binh-minh** [comment]
updated

File: test/core/userdata/0.8/Logs/dynamoLog_84d17b4e-36a6-4789-9f4c-315af23f11c3.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Dynamo log started 6/29/2015 4:05:53 PM
```
Comment (ke-yu) [inline test/core/userdata/0.8/Logs/dynamoLog_84d17b4e-36a6-4789-9f4c-315af23f11c3.txt]: I don't think log file is necessary.

**Comment by nguyen-binh-minh** [comment]
updated

File: src/Libraries/CoreNodeModelsWpf/Converters/SelectionToTextConverter.cs [hunk 3]
```diff
@@ -1,21 +1,30 @@
...
 using System.Globalization;
 using System.Windows.Data;
+using DynamoConversions;
+using Dynamo.Wpf.Properties;
 
-namespace Dynamo.Wpf.Converters
+namespace Dynamo.Controls
 {
-    public class SelectionToTextConverter : IValueConverter
+    public class UnitToTextConverter : IValueConverter
```
Comment (sharadkjaiswal) [inline src/Libraries/CoreNodeModelsWpf/Converters/SelectionToTextConverter.cs]: you may want to change the file name as well. I assume. SelectionToTextConverter is not used anywhere.

File: src/DynamoCoreWpf/Properties/Resources.en-US.resx [hunk 4]
```diff
@@ -1739,4 +1739,85 @@ Do you want to install the latest Dynamo update?</value>
   <data name="PublishPackageViewPackageNameTooltip" xml:space="preserve">
     <value>A unique name for the package.</value>
   </data>
+  <data name="UnitAcres" xml:space="preserve">
```
Comment (sharadkjaiswal) [inline src/DynamoCoreWpf/Properties/Resources.en-US.resx]: Where are these strings being used?

Comment (nguyen-binh-minh) [inline src/DynamoCoreWpf/Properties/Resources.en-US.resx]: it's used inside UnitToTextConverter 's Convert method.

**Pull request merged**